assets/js/src/core/modules/app/theme/utils/themes/default-theme.ts

```diff
@@ -100,6 +100,56 @@ const defaultTheme = {
       colorBgContainer: '#ffffff',
       borderRadiusSM: '4px'
     },
+    Input: {
```

Comment:
Do we need those? Shouldn't these be the default values from ant already?
Because when I get the diff in the theme buddy I don't see them.